hermes/probe/error/error.go

```diff
@@ -22,37 +22,38 @@
 package error
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
 )
 
 // ProbeError is an error that includes an exit status for used within Hermes.
 type ProbeError struct {
+	// TODO(evanSpendlove): Refactor metrics.ExitStatus to use something similar to metrics.APICallStatus.
 	Status metrics.ExitStatus
 	Err    error
 }
 
-// Wrap returns a new ProbeError containing the error passed
+// New returns a new ProbeError containing the error passed
 // with additional context.and the status.
 // Arguments:
 //	- status: pass the exit status associated with this error.
-//	- msg: pass the additional context of the error as a string.
 //	- innner: pass the inner error to be wrapped.
```

Comment:
Nit: I would just call this `err` instead of `inner`.

Reply:
Ack.